index.js

```diff
@@ -77,9 +73,44 @@ class BookLibrary {
   }
 
   clearForm() {
-    this.bookTitle.value = "";
-    this.bookAuthor.value = "";
+    this.bookTitle.value = '';
+    this.bookAuthor.value = '';
   }
 }
 // eslint-disable-next-line no-unused-vars
 const library = new BookLibrary();
+
+const currentDate = new Date();
+const options = {
+  year: 'numeric', month: 'long', day: 'numeric', hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: false,
+};
+const formattedDate = currentDate.toLocaleDateString('en-US', options);
+const displayDate = document.getElementById('date-display');
+displayDate.textContent = formattedDate;
+
+document.addEventListener('DOMContentLoaded', () => {
+  const navItems = document.querySelectorAll('.nav-item');
```

Comment:
- [ ] You did a great job working on this milestone. 💯 However, your project is incomplete, there aren't elements with the `.nav-item` class. Please make sure to double-check that everything is working before submitting your code for review. 

![image](https://github.com/Obimbo07/awesome-books/assets/77069011/3b778f21-b075-49c9-bcc3-ef061c374336)

Please consider fixing this! 🤓